Bottom: c51fca71162992b9721c6a310d694b2bc0fe44e0
Top:    de798b336102184cdabacd4721a7feebe59262b6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-02 14:49:19 +0100

autoroute.c: Temporarily turn on some debugging drawing


---

diff --git a/src/autoroute.c b/src/autoroute.c
index d419031..5c767ea 100644
--- a/src/autoroute.c
+++ b/src/autoroute.c
@@ -92,16 +92,14 @@ RCSID ("$Id$");
 #define ROUTE_VERBOSE
 */
 
-/*
 #define ROUTE_DEBUG
-//#define DEBUG_SHOW_ROUTE_BOXES
+#define DEBUG_SHOW_ROUTE_BOXES
 #define DEBUG_SHOW_EXPANSION_BOXES
-//#define DEBUG_SHOW_EDGES
-//#define DEBUG_SHOW_VIA_BOXES
+#define DEBUG_SHOW_EDGES
+#define DEBUG_SHOW_VIA_BOXES
 #define DEBUG_SHOW_TARGETS
 #define DEBUG_SHOW_SOURCES
-//#define DEBUG_SHOW_ZIGZAG
-*/
+#define DEBUG_SHOW_ZIGZAG
 
 static direction_t
 directionIncrement(direction_t dir)
@@ -4191,7 +4189,7 @@ RouteOne (routedata_t * rd, routebox_t * from, routebox_t * to, int max_edges)
       /* we should never add edges on inactive layer groups to the heap. */
       assert (is_layer_group_active[e->rb->group]);
 #if defined(ROUTE_DEBUG) && defined(DEBUG_SHOW_EXPANSION_BOXES)
-      //showedge (e);
+      showedge (e);
 #endif
       if (e->rb->flags.is_thermal)
 	{
